src/main/resources/suppressions.xml

```diff
@@ -0,0 +1,82 @@
+    <suppress checks="JavadocPackage" files=".*[\\/]examples[\\/].*"/>
+
+    <!-- suppress javadoc check for impl classes and package-info.java -->
+    <suppress checks="JavadocType" files=".*Impl\.java$" />
+    <suppress checks="JavadocStyle" files=".+[\\/]package-info\.java$" />
+
+    <!-- suppress all checks in the generated directories -->
+    <suppress checks=".*" files=".+[\\/]generated[\\/].+\.java"/>
+    <suppress checks=".*" files=".+[\\/]generated-sources[\\/].+\.java"/>
+    <suppress checks=".*" files=".+[\\/]generated-test-sources[\\/].+\.java"/>
+
+    <!-- suppress most all checks expect below-->
+    <suppress checks="^(?!.*(UnusedImports|IllegalImport)).*$" files=".*[\\/]src[\\/]test[\\/].*"/>
+
+    <suppress checks="IllegalImport" files="ProxyServiceStarter.java"/>
+
+    <!-- suppress all checks in the copied code -->
+    <suppress checks=".*" files=".+[\\/]com[\\/]scurrilous[\\/]circe[\\/].+\.java"/>
+
+    <suppress checks=".*" files="MLDataFormats.java"/>
+    <suppress checks=".*" files="BitSetRecyclable.java"/>
+    <!-- pulsar io -->
+    <suppress checks="RegexpSinglelineJava" files="KafkaConnectData.java"/>
+    <suppress checks="LineLength" files="KafkaConnectData.java"/>
+    <suppress checks="RegexpSinglelineJava" files="KafkaConnectSink.java"/>
+    <!-- pulsar io es -->
+    <suppress checks="LineLength" files="ElasticSearchConfig.java"/>
+    <suppress checks="LineLength" files="ElasticSearchClient.java"/>
+    <!-- pulsar io flume -->
+    <suppress checks="RegexpSingleline" files="Application.java"/>
+    <suppress checks="RegexpSinglelineJava" files="AbstractConfigurationProvider.java"/>
+    <suppress checks="RegexpSinglelineJava" files="AbstractZooKeeperConfigurationProvider.java"/>
+    <suppress checks="RegexpSinglelineJava" files="PollingPropertiesFileConfigurationProvider.java"/>
+    <suppress checks="RegexpSinglelineJava" files="EnvVarResolverProperties.java"/>
+    <!-- pulsar io hbase -->
+    <suppress checks="RegexpSinglelineJava" files="HbaseAbstractSink.java"/>
+    <suppress checks="RegexpSinglelineJava" files="HbaseAbstractConfig.java"/>
+    <suppress checks="RegexpSinglelineJava" files="HbaseSinkConfig.java"/>
+    <!-- pulsar io kafka -->
+    <suppress checks="FallThrough" files="KafkaConnectData.java"/>
+    <!-- pulsar io influxdb -->
+    <suppress checks="RegexpSinglelineJava" files="InfluxDBSinkConfig.java"/>
+    <!-- pulsar io kinesis -->
+    <suppress checks="LineLength" files="KinesisSinkConfig.java"/>
+    <suppress checks="LineLength" files="KinesisSink.java"/>
+    <suppress checks=".*" files=".+[\\/]org[\\/]apache[\\/]pulsar[\\/]io[\\/]kinesis[\\/]fbs[\\/].+.java"/>
+    <!-- pulsar io mongo -->
+    <suppress checks="RegexpSinglelineJava" files="MongoSink.java"/>
+    <suppress checks="RegexpSinglelineJava" files="MongoConfig.java"/>
+    <!-- pulsar io rabbitmq -->
+    <suppress checks="RegexpSinglelineJava" files="RabbitMQAbstractConfig.java"/>
```

Comment:
it looks like there's unnecessary content here. Please cleanup

Reply:
The suppressions.xml file is used to specify the check style configuration for the repository[0], and it is consistent with the content in Pulsar[1].

[0]https://github.com/apache/pulsar-java-contrib/pull/2/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R162

[1]https://github.com/apache/pulsar/blob/66e1a06b247cd032872e5ab454ff8c8c6cd98550/pom.xml#L2170

Reply:
well these files are about apache/pulsar repository content. It doesn't make sense to keep these when copying this over to this repository. please remove.